### PR TITLE
luci-app-banip: various optimizations

### PR DESCRIPTION
**Pull request by dibdot**
* add various translation improvements based on the work of @stokito (see #7090)
* use the new logreadbox component for the firewall log as well
* removed obsolete standard Save/Save & Apply buttons in the footer,
   use only the improved/customized "Reload" and "Restart" buttons
* improved the error handling in the try/catch blocks

Signed-off-by: Dirk Brenken <dev@brenken.org>

### CODE REVIEW

File: applications/luci-app-banip/htdocs/luci-static/resources/view/banip/firewall_log.js [hunk 1]
```diff
@@ -1,41 +1,4 @@
...
-					'wrap': 'off',
-					'rows': 25
-				})
-			]));
-	},
-	handleSaveApply: null,
-	handleSave: null,
-	handleReset: null
-});
+return views.LogreadBox(" banIP/", "banIP firewall logs");
```
Comment (stokito) [inline applications/luci-app-banip/htdocs/luci-static/resources/view/banip/firewall_log.js:4]: ` banIP/` it this correct that it starts with space?

Comment (dibdot) [reply applications/luci-app-banip/htdocs/luci-static/resources/view/banip/firewall_log.js:4]: yep, that's intentional/required

File: applications/luci-app-banip/htdocs/luci-static/resources/view/banip/overview.js [hunk 2]
```diff
@@ -564,12 +594,15 @@ return view.extend({
 		o.rmempty = true;
 
 		o = s.taboption('adv_log', form.ListValue, 'ban_loglimit', _('Log Limit'), _('Parse only the last stated number of log entries for suspicious events. To disable the log monitor at all set it to \'0\'.'));
-		o.value('0', _('0 (disable)'));
-		o.value('50', _('50'));
-		o.value('100', _('100 (default)'));
-		o.value('250', _('250'));
-		o.value('500', _('500'));
-		o.value('1000', _('1000'));
+		o.value('0');
```
Comment (stokito) [inline applications/luci-app-banip/htdocs/luci-static/resources/view/banip/overview.js:597]: the change belongs to another PR

File: applications/luci-app-banip/htdocs/luci-static/resources/view/banip/firewall_log.js (hunk 1 above)
Comment (stokito) [reply applications/luci-app-banip/htdocs/luci-static/resources/view/banip/firewall_log.js:4]: looks confusing, but hope it's only for me. Minor style issue: use `'` instead of `"`

File: applications/luci-app-banip/htdocs/luci-static/resources/view/banip/overview.js (hunk 2 above)
Comment (stokito) [reply applications/luci-app-banip/htdocs/luci-static/resources/view/banip/overview.js:597]: I mean, maybe you just squash and merge the previous PR and then rebase the PR on top of that?
There was a lot of changes in the po files itself, I restored many translations

**Review by systemcrash — APPROVED** [review]

**Comment by stokito** [comment]
As minor thing: please don't mention users in commit messages like here:

> add various translation improvements based on the work of @stokito

The GitHub shows notification each time when the commit appears in forks or PRs
